test/services/clients/dissolutionApi.client.test.ts

```diff
@@ -0,0 +1,57 @@
+import axios, { AxiosInstance, AxiosRequestConfig } from 'axios'
+import { assert } from 'chai'
+import sinon from 'sinon'
+
+import { DissolutionApiClient } from 'app/services/clients/dissolutionApi.client'
+
+import { generateAxiosResponse } from 'test/fixtures/axios.fixtures'
+import {
+  generateDissolutionCreateRequest,
+} from 'test/fixtures/dissolutionService.fixtures'
+
+describe('DissolutionApiClient', () => {
+
+  let axiosInstance: AxiosInstance
+  let dissolutionApiUrl: string
+  let client: DissolutionApiClient
+  let postStub: sinon.SinonStub
+
+  const TOKEN = 'some-token'
+  const COMPANY_NUMBER = '12345678'
+  const BODY = generateDissolutionCreateRequest()
+  const RESPONSE = generateAxiosResponse({
+    application_reference_number: '123ABC'
+  })
+
+  beforeEach(() => {
+    axiosInstance = axios.create()
+    postStub = sinon.stub().returns(RESPONSE)
+    axiosInstance.post = postStub
+    dissolutionApiUrl = 'http://apiurl.com'
+
+    client = new DissolutionApiClient(dissolutionApiUrl, axiosInstance)
+  })
+
+  describe('createDissolution', () => {
+    it('should create a dissolution request in the api and return reference number', async () => {
+      const response = await client.createDissolution(TOKEN, COMPANY_NUMBER, BODY)
+
+      const reqUrl: string = `${dissolutionApiUrl}/dissolution-request/${COMPANY_NUMBER}`
+      const reqConfig: AxiosRequestConfig = {
+        headers: {
+          Authorization: 'Bearer ' + TOKEN,
+          'Content-Type': 'application/json',
+          'Accept': 'application/json',
+        }
+      }
```

Comment:
Looks like you are constructing this object just for the purpose of asserts. I wouldn't construct unneccessary objects. Just write individual assert statements targeting each of the different properties:
```
assert.equal(config.header.Authorization, 'Bearer ' + TOKEN)
assert.equal('Content-Type', 'application/json')
assert.equal('Accept', 'application/json')
```